### ISSUE TITLE
error loading folders

### ISSUE THREAD

**Issue by oscarangeli** [nilsteampassnet/TeamPass#1309]
OS: Debian 8 64bit
Apache/2.4.10, PHP 5.6.20, MySQL 5.5.49
TeamPass 2.1.26
Openldap authentication on MS AD domain.

Some users have no problems at all. Other users after login get "Requested JSON parse failed" and the page stands while unsuccessfully loading the folder tree.
The debug console gives "jstree.min.js:2 Uncaught TypeError: this.settings.core.error.call is not a function" on "jstree.min.js".
This happens only beause of shared folder. If only personal folder is enabled, the same user gets no error.

Any hint is welcome :)

**Comment by oscarangeli** [comment]
Investigating the problem I've found that it happens on a user who has rights to access to a subFolder but not to the parent one, when the option "Simplify the Items Tree by removing the Folders the user has no access to" is enabled.

I've found also that if the option "New sub-folder inherits rights from parent folder" is "yes" there's no inheritance and subfolders are created according to user permissions. On the opposite when "no" is selected, the inheritance is correctly set.
